fetch_bringup/package.xml

```diff
@@ -17,10 +17,13 @@
   <exec_depend>diagnostic_aggregator</exec_depend>
   <exec_depend>fetch_description</exec_depend>
   <exec_depend>fetch_drivers</exec_depend>
+  <exec_depend>fetch_navigation</exec_depend>
+  <exec_depend>fetch_open_auto_dock</exec_depend>
   <exec_depend>fetch_teleop</exec_depend>
   <exec_depend>graft</exec_depend>
   <exec_depend>image_proc</exec_depend>
   <exec_depend>joy</exec_depend>
+  <exec_depend>moveit_ros</exec_depend>
```

Comment:
I think we probably depend on `fetch_moveit_config`

https://github.com/fetchrobotics/fetch_ros/blob/melodic-devel/fetch_moveit_config/package.xml